### PR TITLE
chore(plasma-ui): add more performance tests

### PR DESCRIPTION
**Pull request by akhdrv**
:baby_chick: Download canary assets:

[colors--canary.278.3943809752.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.278.3943809752.xml)  
[color_metro_ios-swift--canary.278.3943809752.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_ios-swift--canary.278.3943809752.swift)  
[color_metro_kotlin--canary.278.3943809752.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_kotlin--canary.278.3943809752.kt)  
[color_metro_react-native--canary.278.3943809752.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_react-native--canary.278.3943809752.ts)  
[color_sbermarket_ios-swift--canary.278.3943809752.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.278.3943809752.swift)  
[color_sbermarket_kotlin--canary.278.3943809752.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.278.3943809752.kt)  
[color_sbermarket_react-native--canary.278.3943809752.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.278.3943809752.ts)  
[color_sberprime_ios-swift--canary.278.3943809752.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.278.3943809752.swift)  
[color_sberprime_kotlin--canary.278.3943809752.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.278.3943809752.kt)  
[color_sberprime_react-native--canary.278.3943809752.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.278.3943809752.ts)  
[color_selgros_ios-swift--canary.278.3943809752.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_ios-swift--canary.278.3943809752.swift)  
[color_selgros_kotlin--canary.278.3943809752.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_kotlin--canary.278.3943809752.kt)  
[color_selgros_react-native--canary.278.3943809752.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_react-native--canary.278.3943809752.ts)  
[color_smbusiness_ios-swift--canary.278.3943809752.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_ios-swift--canary.278.3943809752.swift)  
[color_smbusiness_kotlin--canary.278.3943809752.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_kotlin--canary.278.3943809752.kt)  
[color_smbusiness_react-native--canary.278.3943809752.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_react-native--canary.278.3943809752.ts)  
[PlasmaTokensColor--canary.278.3943809752.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.278.3943809752.swift)  
[typo_mage_ios-swift--canary.278.3943809752.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.278.3943809752.swift)  
[typo_mage_kotlin--canary.278.3943809752.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.278.3943809752.kt)  
[typo_mage_react-native--canary.278.3943809752.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.278.3943809752.ts)  
[typo_plasma_ios-swift--canary.278.3943809752.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.278.3943809752.swift)  
[typo_plasma_kotlin--canary.278.3943809752.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.278.3943809752.kt)  
[typo_plasma_react-native--canary.278.3943809752.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.278.3943809752.ts)  
[typo_ruler_ios-swift--canary.278.3943809752.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.278.3943809752.swift)  
[typo_ruler_kotlin--canary.278.3943809752.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.278.3943809752.kt)  
[typo_ruler_react-native--canary.278.3943809752.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.278.3943809752.ts)  
[typo_sage_ios-swift--canary.278.3943809752.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.278.3943809752.swift)  
[typo_sage_kotlin--canary.278.3943809752.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.278.3943809752.kt)  
[typo_sage_react-native--canary.278.3943809752.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.278.3943809752.ts)  
[typo_sbermarket_ios-swift--canary.278.3943809752.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.278.3943809752.swift)  
[typo_sbermarket_kotlin--canary.278.3943809752.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.278.3943809752.kt)  
[typo_sbermarket_react-native--canary.278.3943809752.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.278.3943809752.ts)  
[typo_soulmate_ios-swift--canary.278.3943809752.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.278.3943809752.swift)  
[typo_soulmate_kotlin--canary.278.3943809752.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.278.3943809752.kt)  
[typo_soulmate_react-native--canary.278.3943809752.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.278.3943809752.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-temple@1.122.1-canary.278.3943809752.0
  npm install @salutejs/plasma-ui@1.155.1-canary.278.3943809752.0
  # or 
  yarn add @salutejs/plasma-temple@1.122.1-canary.278.3943809752.0
  yarn add @salutejs/plasma-ui@1.155.1-canary.278.3943809752.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🌚 Missing base report**

Check out current report in job artifacts!

File: packages/plasma-ui/src/components/Card/Card.perftest.tsx [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+    cover: false,
+    coverGradient: true,
+    outlined: true,
+    scaleOnFocus: true,
+    subTitle: 'Потребительский кредит',
+    title: 'до 230 000 ₽',
+    description: 'На 18 месяцев, ставка 13,9%',
+};
+
+export const BasicValue = () => {
```
Comment (KateKate) [inline packages/plasma-ui/src/components/Card/Card.perftest.tsx:32]: может через общий компонент со stories? вынести в общее в один компонент. то копи-паста много

```tsx
export const BasicValue: Story<BasicValueProps> = ({
    outlined,
    coverGradient,
    scaleOnFocus,
    cover,
    subTitle,
    title,
    description,
}) => {
    return (
        <CardBasicValue
            outlined={outlined}
            coverGradient={coverGradient}
            scaleOnFocus={scaleOnFocus}
            cover={cover}
            subTitle={subTitle}
            title={title}
            description={description}
        />
    );
};
```

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-278/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **💀 Missing report, something went wrong**

Check out current report in job artifacts!

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-278/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🔴 Fail**

Check out comparison report in job artifacts!

File: packages/plasma-ui/src/components/Carousel/Carousel.perftest.tsx [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+        initialIndex: 0,
+        axis,
+        delay,
+        longDelay,
+        min: 0,
+        max: items.length - 1,
+    });
+
+    return (
+        <DeviceThemeProvider>
```
Comment (akhdrv) [inline packages/plasma-ui/src/components/Carousel/Carousel.perftest.tsx:73]: убрать лишние обертки

**Review by KateKate — APPROVED** [review]

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-278/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🔴 Fail**

Check out comparison report in job artifacts!

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-278/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-278/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🔴 Fail**

Check out comparison report in job artifacts!

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-278/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-278/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-278/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-278/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-278/temple-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-278/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-278/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-278/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-278/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-278/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-278/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🔴 Fail**

Check out comparison report in job artifacts!

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-278/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-278/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-278/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-278/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-278/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-278/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-278/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-278/temple-storybook/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-278/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-278/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-278/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🔴 Fail**

Check out comparison report in job artifacts!

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out comparison report in job artifacts!

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-278/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-278/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-278/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-278/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-278/temple-storybook/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-278/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🔴 Fail**

Check out comparison report in job artifacts!

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-278/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-278/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-278/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-278/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-278/temple-storybook/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-278/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-278/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-278/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-278/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-278/temple-storybook/

**Review by neretin-trike — APPROVED** [review]

**Review by Yakutoc — APPROVED** [review]

**Pull request merged**